src/binwalk/plugins/hilink.py

```diff
@@ -1,94 +0,0 @@
-#!/usr/bin/env python
-
-import struct
-import string
-import binwalk.core.plugin
-import binwalk.core.compat
-import binwalk.core.common
-try:
-    # Requires the pycrypto library
-    from Crypto.Cipher import DES
-except ImportError as e:
-    DES = None
-
-
-class HilinkDecryptor(binwalk.core.plugin.Plugin):
-
-    '''
-    Plugin to decrypt, validate, and extract Hilink encrypted firmware.
-    '''
-    MODULES = ["Signature"]
-
-    DES_KEY = "H@L9K*(3"
```

Comment:
What's the issue with this removal? Does Microsoft not want binwalk to be useful for analyzing firmware with even minimal obfuscation like this or something?